skimage/data/__init__.py

```diff
@@ -648,6 +650,32 @@ def coins():
     return _load("data/coins.png")
 
 
+def kidney():
+    """Microscopy image of kidney tissue.
+    See `kidney-tissue-fluorescence.tif` entry at
+    https://gitlab.com/scikit-image/data/-/blob/master/README.md#data
```

Comment:
I think there should be more fleshed out information here.

If you want to cross reference, then sphinx will have to fill things in for you.

I think it is important to be rather verbose in
https://scikit-image.org/docs/stable/api/skimage.data.html#skimage.data.brick

rather than sending the user around a multi click hunt.